json/scores.json

```diff
@@ -1,4 +1,189 @@
+            "Issuelink":"https://github.com/opencode18/OpenCode-Collaborative/issues/1",
+            "PRlink":"https://github.com/opencode18/OpenCode-Collaborative/pull/243",
+            "score":"10"
+         },{
+            "name":"Link star and fork to the github repo #268",
+            "Issuelink":"https://github.com/opencode18/OpenCode-Collaborative/issues/268",
+            "PRlink":"https://github.com/opencode18/OpenCode-Collaborative/pull/270",
+            "score":"10"
+         },{
+            "name":"Day5 #86",
+            "Issuelink":"https://github.com/opencode18/CodeStash/issues/86",
+            "PRlink":"https://github.com/opencode18/CodeStash/pull/90",
+            "score":"10"
+         },{
+            "name":"Day6 #96",
+            "Issuelink":"https://github.com/opencode18/CodeStash/issues/96",
+            "PRlink":"https://github.com/opencode18/CodeStash/pull/106",
+            "score":"10"
+         },{
+            "name":"Day7 #109",
+            "Issuelink":"https://github.com/opencode18/CodeStash/issues/109",
+            "PRlink":"https://github.com/opencode18/CodeStash/pull/120",
+            "score":"10"
+         },{
+            "name":"Find more issues in the Website and comment here . #13",
+            "Issuelink":"https://github.com/opencode18/OpenNews/issues/13",
+            "PRlink":"",
+            "score":"10"
+         },{
+            "name":"Generate Github Resume #23",
+            "Issuelink":"https://github.com/opencode18/OpenCode-Collaborative/issues/23",
+            "PRlink":"",
+            "score":"10"
+         },{
+            "name":"star this repository #128",
+            "Issuelink":"https://github.com/opencode18/OpenCode-Collaborative/issues/128",
+            "PRlink":"",
+            "score":"10"
+         },{
+            "name":"Star This Repo #55",
+            "Issuelink":"https://github.com/opencode18/CodeStash/issues/55",
+            "PRlink":"",
+            "score":"10"
+         },{
+            "name":"Make a website for yourself #251",
+            "Issuelink":"https://github.com/opencode18/OpenCode-Collaborative/issues/251",
+            "PRlink":"",
+            "score":"50"
+         },{
+            "name":"Update scores.json #325",
+            "Issuelink":"https://github.com/opencode18/OpenCode-Collaborative/issues/325",
+            "PRlink":"https://github.com/opencode18/OpenCode-Collaborative/pull/344",
+            "score":"10"
+         },{
+            "name":"Day 9 #138",
+            "Issuelink":"https://github.com/opencode18/CodeStash/issues/138",
+            "PRlink":"https://github.com/opencode18/CodeStash/pull/143",
+            "score":"10"
+         },{
+            "name":"Day 10 #150",
+            "Issuelink":"https://github.com/opencode18/CodeStash/issues/150",
+            "PRlink":"https://github.com/opencode18/CodeStash/pull/143",
+            "score":"10"
+         }
+         ],
+         "Total":"170"
+      },
```

Comment:
Looks like you are fixing conflicts in a wrong manner. You should be careful which changes to keep and which not during resolving conflicts. This will duplicate the entry for shivhek25. Please remove Shivhek25 entry.